introduction.Rmd

```diff
@@ -184,6 +184,309 @@ We call the combination of these two steps the __quote and unquote__ pattern. Th
 Most of the time though, we don't need to create blueprints manually. We'll get them by quoting the arguments supplied by users. This gives your functions the same usage and feel as tidyverse verbs.
 
 
+# Do you need tidy eval? {#sec:do-you-need}
+
+In the field of computer science, frameworks like tidy evaluation are known as metaprogramming. Modifying the blueprints of computations amounts to programming the program, i.e. metaprogramming. In other languages, this type of approach is often seen as a last resort because it requires new skills and might make your code harder to read. Things are different in R because of the importance of data masking functions, but it is still good advice to consider other options before turning to tidy evaluation. In this section, we review several strategies for solving programming problems with tidyverse packages.
```

Comment:
* I think you can strike "the field of" and just say "In computer science"
* This paragraph is good, but I wonder if this should open with something more akin to a "TL;DR" (or, perhaps, with some of the concrete _tasks_ that can be done w/out tidy eval that people seem to use it for). @gvwilson might have suggestions for this.

Reply:
You're right about a more practical opening paragraph, I'll add it once the section is finished and I have a better overview.